packages/system/src/utils/general.ts

```diff
@@ -275,16 +275,16 @@ function calculateAppliedPaddings(bounds: Rect, x: number, y: number, zoom: numb
 }
 
 /**
- * Returns a viewport that encloses the given bounds with optional padding.
+ * Returns a viewport that encloses the given bounds with padding.
```

Comment:
padding is not optional 
<img width="285" alt="image" src="https://github.com/user-attachments/assets/8c816e3c-7157-44c7-9610-ef34c4eff760" />